### DISCUSSION TITLE
Problem with login

### DISCUSSION THREAD

**Discussion by tsaki125** [flobz/psa_car_controller#418] [Q&A]
I have PSA controller installed in Home Assistant and i have a problem when i try to login.
In user config tab i insert my credentials and after press the submit button the following message appeared: 

**"Traceback (most recent call last): File "/psa_car_controller/libs/psa/setup/app_decoder.py", line 76, in firstLaunchConfig res_dict = res2.json()["success"] KeyError: 'success' {"code":"495", "message": "Invalid or expired client certificate"}"**

Any help?

**Comment by ricardolobo81** [comment]
Same problem.
I think the version of the PSA_car_controller - HA integration is the 2.6.0, and the last version is 3.0.5.

**Comment by kurim** [comment]
It applies to this issue: https://github.com/flobz/psa_car_controller/issues/416

Please use 3.0.5 in HA, but you need to access through :5000 instead of using ingress

**Reply by PatrikNorrgard** [reply]
Should it update itself or do we need to change to use the psacc-ha-beta repository?

**Reply by kurim** [reply]
To use 3.x.x you need use ha-beta.
Note it will open a diffrent data storage so your old tracked data are not there.

**Reply by ricardolobo81** [reply]
Can you share the link of the ha-beta repository?

Thank you.

**Reply by kurim** [reply]
Sure: https://github.com/flobz/psacc-ha-beta

**Reply by ricardolobo81** [reply]
> Sure: https://github.com/flobz/psacc-ha-beta

Thank you for your help.
